cli/lib/build.js

```diff
@@ -177,12 +149,13 @@ const checkModuleValid = async (execPath)=> {
 
   if (!isAllValid) {
     process.exit(1);
-  } else if (execPath === 'local') {
+  } else if (isLocal) {
     const answer = await inquirer.prompt([
       {
         type: 'confirm',
         name: 'coveredAllModules',
-        message: `Are all modules covered in【${outputModules}】`,
+        message: `Here are the modules【${outputModules}】detected in .env file. If missing module requires to build please update env config and run again. Otherwise press Enter to continue.`,
```

Comment:
build action depends on the ./erda/config.js, so can not just add module to env config.
I suggest that the middle part of the message can be like this: 'If missing module, which needs t rebuild, please update env config by registering module with command "erda-ui setup <module> <port>", and then run again.